tests/frontend/test_kibana_app/cypress/integration/features/filters/pin-filter.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Pin filter
+   
+    As Wazuh user 
+    i want to pin a filter 
+    in order to aplly it across the modules
+  
+   Scenario: The user add a new filer
+    Given The kibana admin user is logged in using xpack and the wazuh logo is displayed
+    When The user navigates to security-events module
+    And The user add a new filter
+    Then The user check filter label is added
+    And The user pin a filter
```

Comment:
this step should be a "When" since the user is performing an action.

and we shouldn't have actions, validations, then actions again and validations again because then we lose focus on the objective of the test.

To solve this you could either:
A- skip the validation:
Given The kibana admin user is logged in using xpack and the wazuh logo is displayed
When The user navigates to security-events module
And The user adds a new filter
And The user pins a filter
And The user checks if the filter is applied across the modules

or 
B- we could write all the previous steps in a precondition and just perform the last actions and validations
Given The kibana admin user is logged in using xpack and the wazuh logo is displayed
And The kibana app is located on security-events module
And The user added a new filter
And The user filter label is added
When The user pins a filter
Then The user check if the filter is applied across the modules

or maybe we could write this in a different way creating a method to synthesize all previous actions